ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/console/DefaultOutputConsole.java

```diff
@@ -89,7 +98,7 @@ public void printText(String text) {
     public void printText(String text, String color) {
         view.print(text, text.endsWith("\r"), color);
 
-        for (ActionDelegate actionDelegate : actionDelegates) {
+    	for (ActionDelegate actionDelegate : actionDelegates) {
```

Comment:
you've to check in the whole file as I see mainly only tabs while spaces are expected.